src/rotp/lang/fr/labels.txt

```diff
@@ -1428,7 +1428,7 @@ SHIP_DESIGN_1ST_SCOUT_NAME        | Éclaireur
 SHIP_DESIGN_1ST_FIGHTER_NAME      | Chasseur
 SHIP_DESIGN_1ST_DESTROYER_NAME    | Contre-torpilleur
 SHIP_DESIGN_1ST_BOMBER_NAME       | Bombardier
-SHIP_DESIGN_1ST_COLONY_NAME       | Colonisie
+SHIP_DESIGN_1ST_COLONY_NAME       | Coloniste
```

Comment:
Pas vraiment un mot français, mais bon ...

Reply:
The intention was: "Colonie", but my fingers got got the hiccups.
Coloniste is nice too :-)